clients/admin-ui/src/features/common/system-data-flow/DataFlowAccordionForm.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  ButtonGroup,
+  Flex,
+  Spacer,
+  Stack,
+  Tag,
+  Text,
+  useDisclosure,
+  useToast,
+} from "@fidesui/react";
+import { FormGuard } from "common/hooks/useIsAnyFormDirty";
+import { GearLightIcon } from "common/Icon";
+import QuestionTooltip from "common/QuestionTooltip";
+import { DataFlowSystemsDeleteTable } from "common/system-data-flow/DataFlowSystemsDeleteTable";
+import DataFlowSystemsModal from "common/system-data-flow/DataFlowSystemsModal";
+import { successToastParams } from "common/toast";
+import { Form, Formik } from "formik";
+import React, { useEffect, useMemo, useState } from "react";
+
+import {
+  useGetAllSystemsQuery,
+  useUpdateSystemMutation,
+} from "~/features/system";
+import { DataFlow, System } from "~/types/api";
+
+const defaultInitialValues = {
+  dataFlowSystems: [] as DataFlow[],
+};
+
+export type FormValues = typeof defaultInitialValues;
+
+type DataFlowAccordionItemProps = {
+  isIngress?: boolean;
+  system: System;
+  isSystemTab?: boolean;
+};
+
+export const DataFlowAccordionForm = ({
+  system,
+  isIngress,
+  isSystemTab,
+}: DataFlowAccordionItemProps) => {
+  const toast = useToast();
+  const flowType = isIngress ? "Source" : "Destination";
+  const pluralFlowType = `${flowType}s`;
+  const dataFlowSystemsModal = useDisclosure();
+  const [updateSystemMutationTrigger] = useUpdateSystemMutation();
+
+  const { data } = useGetAllSystemsQuery();
+  const systems = useMemo(() => data || [], [data]);
+
+  const initialDataFlows = useMemo(() => {
+    const dataFlows = isIngress ? system.ingress! : system.egress!;
+    const systemFidesKeys = systems ? systems.map((s) => s.fides_key) : [];
+
+    return dataFlows.filter((df) => systemFidesKeys.includes(df.fides_key));
```

Comment:
This filters out returned data flows that doesn't have a fides key that's tied to an active system. Right now we're only configuring data flows for systems so it doesn't make sense to factor them in.

The pixie scan was returning and `external` flow that had no system tied to it.